app/models/poa.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+require 'common/models/base'
+
+# POA model
+class Poa < Common::Base
```

Comment:
What would you think about using the long version: PowerOfAttorney ?

Reply:
I have updated this to the explicit name.